sbt_common/messaging/src/main/scala/uk/ac/wellcome/messaging/sqs/SQSStream.scala

```diff
@@ -43,6 +43,7 @@ class SQSStream[T] @Inject()(actorSystem: ActorSystem,
           () => readAndProcess(streamName, message, process))
       }
       .map { m =>
+        println(s"Deleting message ${m.getMessageId}")
```

Comment:
I don't think we should be using println. It gets ordered in a different way than logged strings and printed strings give you less information.
Is there a reason to not use sl4j logging here?

Reply:
I’d forgotten we had it! Will fix.